doc/10-icinga-template-library.md

```diff
@@ -751,6 +768,7 @@ ldap_warning_entries	| **Optional.** Number of found entries to result in warnin
 ldap_critical_entries	| **Optional.** Number of found entries to result in critical status.
 ldap_timeout		| **Optional.** Seconds before connection times out (default: 10).
 ldap_verbose		| **Optional.** Show details for command-line debugging (disabled by default)
+ldap_extra_opts   | **Optional.** Read extra plugin options from an ini file.
```

Comment:
Similar hare.

Reply:
this one's all over the place using with tabs and spaces.. hope I fixed it to your satisfaction :)